ci/dockerfiles/builder/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM analoglabs/base-ci-linux:latest
+
+WORKDIR /chain-connectors
+COPY . /chain-connectors
+
+RUN set -eux && \
+    dpkgArch="$(dpkg --print-architecture)"; \
+    case "${dpkgArch##*-}" in \
+        amd64) targetArch='x86_64-unknown-linux-musl' ;; \
+        armhf) targetArch='armv7-unknown-linux-musleabi' ;; \
+        arm64) targetArch='aarch64-unknown-linux-musl' ;; \
+        i386) targetArch='i686-unknown-linux-musl' ;; \
+        *) echo >&2 "unsupported architecture: ${dpkgArch}"; exit 1 ;; \
+    esac; \
+    cargo build \
+      --locked \
+      --release \
+      --target ${targetArch} \
+      -p rosetta-server-bitcoin \
+      -p rosetta-server-ethereum \
+      -p rosetta-server-polkadot \
+      -p rosetta-server-astar && \
+    mkdir -p ./bin && \
```

Comment:
This is the `cache` image, it build everything in one shot (which is faster), then we just copy the binaries to the specific images.

Reply:
this is faster for ci maybe. it's not faster for local rebuilds.